loader/gamedll.cpp

```diff
@@ -131,12 +165,14 @@ mm_DetectGameInformation()
 			lptr = cur_path;
 		}
 
+		const char *pRelPath = is_source2 ? "../../" : "";
```

Comment:
I don't think that's what we want here since we're dealing with relative paths for comparison.